src/Leaderboard.py

```diff
@@ -109,6 +109,7 @@ def reportMatch(player: Union[Member, str], whoWon: Team, adminOverride: bool =
 
     # report confirmed
     matchMMR = Points.calculateMMR(match)
+    mmr_multiplier = getMMRMultiplier()
```

Comment:
Would it make more sense to do the mmr multiplier logic inside of `Points` instead of here?

Reply:
I'm not sure what you mean by this. This calls DataFiles to get the current MMR multiplier.

Reply:
I'm saying get the mmr multiplier and perform the mmr * mmr_multiplier inside of Points.calculateMMR instead of here in Leaderboard.

Reply:
Also, quick thought, we don't want players to lose more MMR when there's a multiplier do we? So shouldn't it only apply to the winning team?

Reply:
Well, it does only apply to the winning team if you look below where you're commenting on.  The reason I did it this way, is so that I can have the match MMR separate from what the result is if you factor in the multiplier.